src/index.ts

```diff
@@ -21,7 +21,7 @@ app.get('/', (req, res) => {
   res.sendFile('views/index.html', { root: process.cwd() })
 })
 
-function errorHandler(err: Error, req: Request, res: Response, next: NextFunction) {
+function errorHandler(err: Error, req: Request, res: Response) {
```

Comment:
Did you test whether this error handler still works?
As far as I know, express require error handlers to always take 4 arguments, even if not all are used.

https://expressjs.com/en/guide/error-handling.html